global_logger.py

```diff
@@ -1,23 +1,174 @@
-import logging
 import logging.handlers
 import os
+import time
+from enum import auto, Enum
+from functools import wraps
+from typing import Optional
 
+# Set up logging level constants
+CRITICAL = logging.CRITICAL  # 50
+ERROR = logging.ERROR  # 40
+WARNING = logging.WARNING  # 30
+INFO = logging.INFO  # 20
 VERBOSE = 15
-verbose_level = 'VERBOSE'
-log_file = './logs/logon_server.log'
+DEBUG = logging.DEBUG  # 10
```

Comment:
I figured it would make sense to just have all the level constants accessible from this global_logger module.

If you have `from global_logger import *` you'll have all of these at your disposal.